### PR TITLE
Fix download retry helper

### PR DESCRIPTION
**Pull request by vsafonkin**
# Description
Bug fixing: incorrect flag for `curl`  

#### Related issue:
#1066 

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/scripts/helpers/install.sh [hunk 1]
```diff
@@ -12,11 +12,17 @@ download_with_retries() {
     local DEST="${2:-.}"
     local NAME="${3:-${URL##*/}}"
 
+    if [ $4 ]; then
```
Comment (miketimofeev) [inline images/linux/scripts/helpers/install.sh:15]: maybe use something like this to stay on a safe side?
if [ $4="-compressed" ]

File: images/linux/scripts/installers/pypy.sh [hunk 2]
```diff
@@ -72,7 +72,7 @@ function InstallPyPy
 
 # Installation PyPy
 uri="https://downloads.python.org/pypy/"
-download_with_retries $uri "/tmp" "pypyUrls.html"
+download_with_retries $uri "/tmp" "pypyUrls.html" --compressed
```
Comment (miketimofeev) [inline images/linux/scripts/installers/pypy.sh:75]: do we really need these `--`?

File: images/linux/scripts/helpers/install.sh (hunk 1 above)
Comment (vsafonkin) [reply images/linux/scripts/helpers/install.sh]: Done

File: images/linux/scripts/installers/pypy.sh (hunk 2 above)
Comment (vsafonkin) [reply images/linux/scripts/installers/pypy.sh]: Fixed

**Pull request merged**